src/db/db_bench.cc

```diff
@@ -495,6 +495,14 @@ class Benchmark {
     Open();
   }
 
+  void PrintFilterStats()
```

Comment:
Could we move this to the DB implementation? For example, in db_bench.cc there is the function:

  void PrintStats(const char* key) {
    std::string stats;
    if (!db_->GetProperty(key, &stats)) {
      stats = "(failed)";
    }
    fprintf(stdout, "\n%s\n", stats.c_str());
  }

which basically prints out different stats of the underlying db (db_). Could we extend this code path to include a new property "filters" and then use PrintStats to print them? I know its a bit more work, but it would work more cleanly with the existing code.

Reply:
Thanks for the review. I'm working on this.

Reply:
I've made this change. Please let me know if anything needs to be changed.